packages/sdk/lib/browser.ts

```diff
@@ -2,10 +2,42 @@
 import {
   Itly as ItlyBase,
   CallOptions,
+  Environment,
   Event,
   LoadOptions,
+  Logger,
+  Loggers,
+  Options,
+  Plugin,
+  PluginCallOptions,
+  PluginLoadOptions,
   Properties,
+  Validation,
+  ValidationResponse,
 } from './base';
+import {
+  RequestLoggerPlugin,
+  RequestLogger,
+  ResponseLogger,
+} from './internal/RequestLogger';
+
+export {
+  Options,
```

Comment:
@falconandy I think to make this work with codegen we need to export `ItlyBrowser` https://bitbucket.org/seasyd/analytics_tools/src/e3915adfd453ad616a305de14391102e8bc9f57c/sdk-examples/browser-javascript-v3/src/itly/index.js#lines-11